shiipou/simplon-2024-brief-06-TooManyChoco
# 6 implmentation du front du dtails dun event

## About this PR



## Technical highlight/advice


## How to test my changes

- Access react app file
cd react-app
- install the npm files
npm i
- run the servier locally
npm start

it should be looking like this : 

![image](https://github.com/shiipou/simplon-2024-brief-06-TooManyChoco/assets/101038977/10ac8f53-e250-4ed1-9f64-b649da5d7f33)


## Checklist

### I included unit tests that cover my changes

### I added/updated the documentation about my changes
